subprojects/packagefiles/tree-sitter/meson.build

```diff
@@ -0,0 +1,17 @@
+project('tree-sitter', 'c')
+
+tree_sitter_lib = static_library('tree-sitter-runtime', [
```

Comment:
I would almost say remove the `_lib` suffix for this variable. Maybe others disagree

Reply:
sure